rmgpy/molecule/fragment.py

```diff
@@ -950,29 +945,52 @@ def sliceitup_arom(self, molecule, size_threshold=None):
+                        if all(sum(1 for atom in mol.GetAtoms() if atom.GetAtomicNum() == 6) >= size_threshold for mol in mol_set):
+                            if len(mol_set) == 2:
+                                frag1 = Chem.MolToSmiles(mol_set[0])
+                                frag2 = Chem.MolToSmiles(mol_set[1])
+    
+                                frag1_R = frag1.count("Na")
+                                frag1_L = frag1.count("K")
+                                frag2_R = frag2.count("Na")
+                                frag2_L = frag2.count("K")
+    
+                                if frag1_R > frag2_R and frag1_L <= frag2_L:
+                                    frag1_smi = frag1.replace("*", "L")
+                                    frag2_smi = frag2.replace("*", "R")
+                                elif frag1_L > frag2_L and frag1_R <= frag2_R:
+                                    frag1_smi = frag1.replace("*", "R")
+                                    frag2_smi = frag2.replace("*", "L")
+                                elif frag2_L > frag1_L and frag2_R <= frag1_R:
+                                    frag1_smi = frag1.replace("*", "L")
+                                    frag2_smi = frag2.replace("*", "R")
+                                elif frag2_R > frag1_R and frag2_L <= frag1_L:
+                                    frag1_smi = frag1.replace("*", "R")
+                                    frag2_smi = frag2.replace("*", "L")
+                                elif randint(0,1)==1:
+                                    frag1_smi = frag1.replace("*", "L")
+                                    frag2_smi = frag2.replace("*", "R")
                                 else:
-                                    frag_smi = frag.replace("*", "L")
-                            frag_list.append(frag_smi)
-                        break
-                    else:
-                        # turn to next matched_atom_map
+                                    frag1_smi = frag1.replace("*", "R")
+                                    frag2_smi = frag2.replace("*", "L")
+    
+                                frag_list = [frag1_smi, frag2_smi]
+    
+                            elif len(mol_set) > 2: # means it cut into 3 fragments
+                                frag_list = []
+                                for ind, rdmol in enumerate(mol_set):
+                                    frag = Chem.MolToSmiles(rdmol)
+                                    if frag.count("*") > 1:
+                                        frag_smi = frag.replace("*", "R")
+                                    else:
+                                        frag_smi = frag.replace("*", "L")
+                                    frag_list.append(frag_smi)                          
+                                break
+                        else:
+                            # turn to next matched_atom_map
+                            continue
+                    except:
```

Comment:
Can we catch the error explicitly and throw an error message?

Reply:
I'm actually curious, what exceptions are we expecting here?

Reply:
I looked into this, and it actually isn't necessary. I must have put it in while troublshooting.